src/main/java/net/minecraftforge/common/brewing/IBrewingRecipe.java

```diff
@@ -19,9 +19,15 @@
 
 package net.minecraftforge.common.brewing;
 
+import net.minecraft.world.Container;
 import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.crafting.Recipe;
+import net.minecraft.world.item.crafting.RecipeType;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.entity.BrewingStandBlockEntity;
 
-public interface IBrewingRecipe {
+public interface IBrewingRecipe extends Recipe<Container>
```

Comment:
Shouldn't just base this off of the existing brewing recipe. Should be implemented using the recipe system.

Reply:
that's not really possible because my goal is to still support non vanilla brewing stands (via BrewingRecipeRegistry) which is not possible with only the Container as a parameter

Reply:
Sure it is? A container can be created based off of any inventory. The container is only used to check the data, which could be the user's inventory, a block's inventory, etc. These can be wrapped.

Reply:
actually a container is what was the inventory before the change to mojmaps